db/migrate/20220401161642_add_refs_to_likes.rb

```diff
@@ -0,0 +1,9 @@
+class AddRefsToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+    add_reference :likes, :post, null: false, foreign_key: true
+    rename_column :likes, :user_id, :author_id
+    add_index :likes, :user
+    add_index :likes, :post
```

Comment:
- [ ] **[OPTIONAL]** I suggest generating a new migration file and using the newly generated migration to rename your table column instead of adding it here.